src/actions/receive-profile.js

```diff
@@ -678,14 +698,14 @@ if (typeof window === 'object' && window.requestIdleCallback) {
   requestIdleCallbackPolyfill = callback => setTimeout(callback, 0);
 }
 
-class ColascedFunctionsUpdateDispatcher {
-  _updates: FunctionsUpdatePerThread;
+class ColascedSymbolicationStepDispatcher {
```

Comment:
This class is also a bit intimidating, and could use with some docs, even if it was just a sentence describing why it's here.